docs-react-native/react-native/docs/ios/remote-notification-support.md

```diff
@@ -68,7 +68,17 @@ At this point everything should still be running normally. This is the final ste
 Before, moving to the next step, run the app and check it builds successfully – make sure you have the correct target selected. 
 
 ## Edit the payload
-Now everything is setup in your app, you can alter your notification payload to include `notifee_options`:
+Now everything is setup in your app, you can alter your notification payload in two ways:
+
+1. When sending from your BE (either via REST api or firebase admin sdk)
```

Comment:
Most people integrate via firebase, but really, on iOS at least it could be anything that delivers to the device eventually, so making it a little more generic

```suggestion
1. When creating the message contents for sending, in your back-end
```